### PR TITLE
Day 1

### PR DESCRIPTION
**Pull request by samradhshrma**
Day1 pull Request

### CODE REVIEW

File: ComparetheTriplets.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+     * The function accepts following parameters:
+     *  1. INTEGER_ARRAY a
+     *  2. INTEGER_ARRAY b
+     */
+
+    public static List<int> compareTriplets(List<int> a, List<int> b)
+    {
+        Int32 length = a.Count;
+        int alice=0;
+        int bob=0;
```
Comment (narayanagunturi) [inline ComparetheTriplets.cs]: use var keyword instead of explicit data types such as Int32 or int

File: ComparetheTriplets.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+     *
+     * The function is expected to return an INTEGER_ARRAY.
+     * The function accepts following parameters:
+     *  1. INTEGER_ARRAY a
+     *  2. INTEGER_ARRAY b
+     */
+
+    public static List<int> compareTriplets(List<int> a, List<int> b)
+    {
+        Int32 length = a.Count;
```
Comment (narayanagunturi) [inline ComparetheTriplets.cs]: Why Int32 and not int?

File: ComparetheTriplets.cs [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+              alice=1+alice;
+          }
+          else if(a[i]<b[i]){
+              bob=1+bob;
+          }
+      }
+      
+      List<int> result = new List<int>();
+result.Add(alice);
+result.Add(bob);
```
Comment (narayanagunturi) [inline ComparetheTriplets.cs]: please maintain proper indentation

File: SimpleArraySum.cs [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+using System.Text.RegularExpressions;
+using System.Text;
+using System;
+
+class Result
+{
+    public static int simpleArraySum(List<int> ar)
+    {
+      int sum = 0;
+      Int32 length = ar.Count;
```
Comment (narayanagunturi) [inline SimpleArraySum.cs]: No need to use extra variable. You can directly use ar.Count in for loop

File: SimpleArraySum.cs [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+using System.Runtime.Serialization;
+using System.Text.RegularExpressions;
+using System.Text;
+using System;
+
+class Result
+{
+    public static int simpleArraySum(List<int> ar)
+    {
+      int sum = 0;
```
Comment (narayanagunturi) [inline SimpleArraySum.cs]: Use var keyword instead of explicit data types int

File: SimpleArraySum.cs [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+      Int32 length = ar.Count;
+    for (int i = 0; i < length; i++)
+{
+    sum += ar[i];
+
+}
+return sum;
+}
+}
```
Comment (narayanagunturi) [inline SimpleArraySum.cs]: please maintain proper indentation

File: SolveMeFirst.cs [hunk 7]
```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+class Solution {
+
+    static int solveMeFirst(int a, int b) { 
+      int c=a+b;
```
Comment (narayanagunturi) [inline SolveMeFirst.cs]: No need to declare an extra variable. you can return directlya+b. one extra line of code will mean extra maintenance. Please keep this in mind

File: ComparetheTriplets.cs (hunk 1 above)
Comment (samradhshrma) [reply ComparetheTriplets.cs]: @narayanagunturi I have resolved this problem.

File: ComparetheTriplets.cs (hunk 3 above)
Comment (samradhshrma) [reply ComparetheTriplets.cs]: @narayanagunturi I have fixed the problem stated as stated in comment

File: SimpleArraySum.cs (hunk 4 above)
Comment (samradhshrma) [reply SimpleArraySum.cs]: @narayanagunturi I have resolved this problem.

File: SimpleArraySum.cs (hunk 6 above)
Comment (samradhshrma) [reply SimpleArraySum.cs]: @narayanagunturi I have resolved this problem.

File: SolveMeFirst.cs (hunk 7 above)
Comment (samradhshrma) [reply SolveMeFirst.cs]: @narayanagunturi I have resolved this problem and will keep in mind next time.

File: SimpleArraySum.cs (hunk 5 above)
Comment (samradhshrma) [reply SimpleArraySum.cs]: @narayanagunturi  I have resolved this problem.

**Review by narayanagunturi — APPROVED** [review]

**Pull request merged**